image_server/image_server.py

```diff
@@ -0,0 +1,141 @@
+import matplotlib
```

Comment:
I think this image server is probably best served from another repository rather than inside the analytics library, because this will introduce flask dependencies that really don't have anything to do with analyzing data.

We can make a branch of the https://github.com/platypusllc/cloud private repository where we can put this image server, it uses platypus/analytics as a dependency so you can still import `platypus.io.logs`.